src/exe/udp_client_test/main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <sputter/net/port.h>
```

Comment:
So, for a test I actually suggest adding a new test case under `sputter/src/test/sputter/net` instead. There's a fixed memory container test under `sputter/src/test/sputter/containers` that should hopefully serve as a reasonable reference!

Reply:
BTW if you'd rather not fuck around with cmake to make this happen, I don't mind making that change. 

At a glance I'm not actually sure why the setup you have here isn't working-- maybe having a chance to mess around with this locally will help me figure it out??

Reply:
This `udp_client_test` is poorly named - the resulting binary tests our strategy for UDP hole punching with another peer on a different host. 

Though we absolutely need a test for the UDPPort code.

Reply:
Oh! Got it, makes sense. Yeah in that case, a quick renaming makes sense.